Automattic/wp-calypso
# Marketplace: Cleanup unused purchase flow state

#### Changes proposed in this Pull Request

This PR cleans up some marketplace unused state:

* `resetPurchaseFlow` action creator - unused since its creation in #53748
* `siteTransferStateChange` action creator - unused since its last usage removal in #59065
* `siteTransferWithPluginInstallTriggered` action creator - unused since its last usage removal in #59065
* `getPrimaryDomainCandidate` selector - unused since its creation in #53037
* `getIsProductSetupComplete` selector - unused since its last usage removal in #59065
* `getHasProductSetupError` selector - unused since its last usage removal in #59065
* `getIsPluginInformationLoaded ` selector - unused since its last usage removal in #59065
* Related unused reducer pieces, type pieces, schema and tests.

#### Testing instructions

Verify Calypso builds without errors, removed code is not in use and tests pass.
